src/domain_bridge/component_manager.cpp

```diff
@@ -22,7 +22,15 @@ namespace domain_bridge
 rclcpp::NodeOptions
 ComponentManager::create_node_options(const std::shared_ptr<LoadNode::Request> request)
 {
-  auto options = rclcpp_components::ComponentManager::create_node_options(request);
+  rclcpp::NodeOptions options;
+
+  // TODO(@fujitatomoya): remove this try-catch block after ComponentManager::create_node_options
+  // is fixed to avoid throwing exceptions for unknown parameters in the base class.
+  //   https://github.com/ros2/rclcpp/pull/2723
```

Comment:
i think it takes some time until https://github.com/ros2/rclcpp/pull/2723 is synced and released in rolling distro, so ended up having the temporary work-around to make domain_bridge workflow and rpr job green.

Reply:
You wouldn't want to wait until the rclcpp PR is merged and released?

Reply:
i think it could be merged soon, but releasing can take time. besides i do not think this work-around does not bring any downsides. or we should wait it out?

Reply:
I was just wondering if there was any reason _not_ to wait, but I don't mind.